scout/server/blueprints/cases/views.py

```diff
@@ -1108,3 +1142,10 @@ def host_image_aux(institute_id, case_name, image, imgstr):
     img_path = abs_path + "/" + imgstr
     LOG.debug("Attempting to send {}/{}".format(img_path, image))
     return send_from_directory(img_path, image)
+
+
+def _generate_csv(header, lines):
```

Comment:
This works, and is readable to me, but if you feel like changing anything, consider being systematic about where you do the comma joining, quotation etc. Right now its a little different for header, lines and then end lines get added by this function.

Reply:
Thanks for the review. In general the only fields that need quotation are those which might contain commas (that are also the col separators). You have a point about consistency, but let's address next time we need this function to export something else!